common/assertion.h

```diff
@@ -80,11 +80,9 @@ static inline void debugger_breakpoint(void)
 // Assert Helper
 //--------------------------------------------------------------------+
 //#ifndef _TEST_
-//  #define ASSERT_MESSAGE(format, ...)\

```

Comment:
GCC does not like `//`-style comment lines that end with a backslash. You get a nasty warning under the `-Wcomments` flag (which is enabled by `-Wall`).